.travis.yml

```diff
@@ -2,4 +2,3 @@ language: node_js
 node_js:
   - "node"
   - "lts/*"
-  - "0.10"
```

Comment:
Node.js v0.10.x is completely out of date and there is no need to be able to build the project with it anymore. In fact, this version was already removed on the `master` branch [a few months ago](https://github.com/Kashoo/synctos/commit/dafa61aab3c8202e68fc73a829d9fb451afd6b71#diff-354f30a63fb0907d4ad57269548329e3). This change should hopefully put an end to the intermittent spurious build failures on this branch.